src/main/java/com/sleekbyte/tailor/utils/Configuration.java

```diff
@@ -68,8 +68,58 @@ public boolean debugFlagSet() throws CliArgumentParserException {
         return cliArgumentParser.debugFlagSet();
     }
 
+    /**
+     * Collects all rules enabled by default and then filters out rules according to CLI options
+     * and YamlConfiguration file.
+     *
+     * @return list of enabled rules after filtering
+     * @throws CliArgumentParserException if rule names specified in command line options are not valid
+     */
     public Set<Rules> getEnabledRules() throws CliArgumentParserException {
```

Comment:
Are there existing tests for this method? I see some tests for CLI options in `CliOptionsTest`, but it would be good to have tests for this method as well.